src/main/resources/config/liquibase/changelog/20210930113000_changelog.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="schlesinger" id="20210930113000">
+        <addColumn tableName="answer_post">
+            <column name="resolves_post" type="BIT(1)" defaultValue="0">
+                    <constraints nullable="false"/>
```

Comment:
copy the data of the other column (potentially in a sql script)

Reply:
👍 done in 1fcfa41